crates/shared/src/price_estimation.rs

```diff
@@ -41,16 +40,15 @@ use {
     thiserror::Error,
 };
 
-#[derive(Copy, Clone, Debug, Hash, Eq, PartialEq, ValueEnum)]
-#[clap(rename_all = "verbatim")]
+#[derive(Copy, Clone, Debug, Hash, Eq, PartialEq)]
 pub enum PriceEstimatorType {
-    Baseline,
-    Paraswap,
-    ZeroEx,
-    Quasimodo,
-    OneInch,
-    Yearn,
-    BalancerSor,
+    Baseline(H160),
+    Paraswap(H160),
+    ZeroEx(H160),
+    Quasimodo(H160),
+    OneInch(H160),
+    Yearn(H160),
+    BalancerSor(H160),
```

Comment:
Every type of estimator will have an address (right?), so maybe this should be split into `PriceEstimator = Address + PriceEsimatorKind`.

Reply:
It's also obnoxious to have to match on this value all the time.

Reply:
The intention is indeed to only allow solvers to become estimators, and so every estimator will have an address. Note however that this means we should create an account for Quasimodo again, as it is the only "solver" that is actually only used as an estimator. (we had delisted it as a solver when we did the pks rotation).

Reply:
Ok, so the `--price-estimators` field will be deprecated at some point.